src/Components/ReadingListBook/ReadingListBook.css

```diff
@@ -0,0 +1,35 @@
+.card-image {
+  border: solid 3px whitesmoke;
+  width: 90%;
+  height: 40em;
+  margin-bottom: 1em;
+}
+
+.card-image:hover {
+  transform: scale(1.1);
+  box-shadow: 15px 0px 15px 0px #C09081;
+}
+
+.book-card {
+  padding: 2em;
+}
+
+.remove-book {
+  margin-top: 0.6em;
+  padding: 0.5em;
+  font-size: 26px;
+  color:  #C09081;
+  font-family: 'Amatic SC', cursive;
+  font-weight: bold;
+  background-color: transparent;
+  border: solid 1px  #C09081;
+  border-radius: 0.3em;
+  box-shadow: 2px 2px 15px  #C09081;
+}
+
+.remove-book:hover {
+  background-color: teal;
+  color: white;
+  border: solid 1px  white;
+  box-shadow: 2px 2px 15px  whitesmoke;
+}
```

Comment:
I really like this hover style. Should we add this to the home page?

Reply:
I think so!! Would we have to change the whole card to be clickable then?

Reply:
Definitely agree!